### DISCUSSION TITLE
Qué hacer si te encuentras un fallo al estar haciendo tests

### DISCUSSION THREAD

**Discussion by laurolmer** [perealberto/decide-single-zambrano#90] [Q&A]
Hola chicos, abro esta discusión de cara a que, hablando con @MarioArocaPaez ha comentado que se había topado con un fallo dentro de la funcionalidad de register ya que en la API no funciona bien el mensaje de error, cuestionándose si debe solucionarlo en su actual rama /test o si debería crear una nueva rama que sea /fix, arreglar el fallo en ella, y seguir con los tests en su rama /test.

Yo en lo personal voto que lo haga en la misma rama /test para ahorrar tiempo, ya que es un fallo minúsculo y creo que en parte entra dentro de la funcionalidad de las ramas /test. ¿Qué opináis?

**Comment by perealberto** [comment]
Buenas noches compañeros! Yo personalmente pienso que es mejor que se cambie de rama para dejar /test únicamente para lo que se ideó (los tests). Si es tan minúsculo, podría hacerse dentro de la rama /feat para evitar que se haga una nueva rama /fix.

@laurolmer @MarioArocaPaez

**Comment by silvglad** [comment]
Yo como os comenté, me parece correcto fixearlo dentro de la rama de test si es un problema menor, ya que por algo se están comprobando las funcionalidades y, si un error aparece, tiene sentido intentar arreglarlo dentro de la misma rama. Si estuviésemos hablando de problemas mayores, sí que opino que se debería abrir una nueva issue y rama de hotfix
